src/Compilers/VisualBasic/Test/Semantic/Compilation/CompilationAPITests.vb

```diff
@@ -1335,7 +1409,7 @@ BC2014: the value '_' is invalid for option 'RootNamespace'
 
             Assert.True(type.IsAnonymousType)
             Assert.Equal(2, type.GetMembers().OfType(Of IPropertySymbol).Count())
-            Assert.Equal("<anonymous type: m1 As Integer, m2 As Boolean>", type.ToDisplayString())
+            Assert.Equal("<anonymous type: Key m1 As Integer, Key m2 As Boolean>", type.ToDisplayString())
```

Comment:
Yes.  that's intentional and desired.  Note: we have not shipped this API yet.  It was created for the IDE and the IDE would like it so that calling with the same arguments for VB and C# produces the same result.